src/Assets/LightSkyDefense/Scripts/Credit.cs

```diff
@@ -17,7 +23,11 @@ private void OnHandHoverBegin(Hand hand)
     {
         var player = Player.instance.gameObject;
         var creditOwner = player.GetComponent<CreditOwner>();
```

Comment:
I would inline the player variable, it's only used on the next line 😕.

```suggestion
        var creditOwner = Player.instance.gameObject.GetComponent<CreditOwner>();
```

_Note: `Commit suggestion` doesn't work on multiple lines, you need to fix this yourself 😞_